Cabal/Cabal.cabal

```diff
@@ -315,6 +315,7 @@ library
     Distribution.Compat.Exception
     Distribution.Compat.Graph
     Distribution.Compat.Internal.TempFile
+    Distribution.Compat.Prelude
```

Comment:
btw, if we want to discourage `Setup.hs` using that module, we could add a reexporting exposed  module `Distribution.Compat.Prelude.Internal`, and hide its documentation in Haddock....